.github/workflows/ci.yml

```diff
@@ -16,13 +16,13 @@ on:
   workflow_dispatch:
 
 env:
-  VCPKG_COMMIT: "6f7ffeb18f99796233b958aaaf14ec7bd4fb64b2"
+  VCPKG_COMMIT: "c8696863d371ab7f46e213d8f5ca923c4aef2a00"
```

Comment:
You wrote in one of the commits that this was the 2024.01 version, but it seems to be the hash of the [2023.12](https://github.com/microsoft/vcpkg/releases/tag/2023.12.12) version

Reply:
I was referring to vcpkg not abseil in the commit message

Reply:
But this is also the vcpkg commit hash pointing to 2023.12?

Reply:
Ahh yea my bad! I was just looking at our main repo but I missed that it was a revert commit. But in any case this matches that version :) still fine